Firmware/ultralcd.cpp

```diff
@@ -791,11 +792,9 @@ void lcd_status_screen()                          // NOT static due to using ins
 	if (lcd_status_update_delay)
 		lcd_status_update_delay--;
 	else
-		lcd_draw_update = 1;
```

Comment:
I'm not so sure this change is needed. Before a lcd_draw_update event from elsewhere would re-render the status screen instantly. Now, a pending lcd update has to wait for the lcd_status_update_delay to expire, which will surely introduce some annoying delays. I'd keep it here how it was before

Reply:
I see what you mean. Do you agree ca8d47a6da33e74b89a29b5c0a6bb1decac951bb is sufficient? I don't see a reason why `lcd_draw_update` needs to be assigned a value in `lcd_status_screen()`. But if it is set outside `lcd_status_screen()` then we can render the screen immediately.

Reply:
Yeah, that should work